SU-HSDO/suhumsci
# HSD8-418 Allow user to hide block title

# READY FOR REVIEW

# Summary
- Allow users to hide block titles

# Needed By (Date)
- 10/24

# Urgency
- high

# Steps to Test

1. checkout this branch
1. composer update
1. sync archaeology
1. edit one of the blocks in the footer
1. check the box "Hide Block Description"
1. save and validate block title is gone. 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
